modelskill/comparison/_comparison.py

```diff
@@ -42,6 +44,31 @@
     from ._collection import ComparerCollection
 
 
+class Scoreable(Protocol):
+    # TODO should this be able to return None?
+    def score(self, metric: str | Callable, **kwargs) -> Dict[str, float] | None:
+        ...
+
+    def skill(
+        self,
+        by: Optional[Union[str, List[str]]] = None,
+        metrics: Optional[List[str]] = None,
+        **kwargs,
+    ) -> Optional[SkillTable]:
```

Comment:
Remove optional? it is a remain from when the filtering was done inside the Comparer...